proposals/lip-0071.md

```diff
@@ -28,20 +28,20 @@ In the current [Lisk protocol][lip-0042], the rewards are the same for all activ
 
 ### Rewards Proportional to the Validators Weight
 
-* This module distributes the remaining stake rewards i.e. `stakeRewardActiveValidators` proportional to the validators weight. It is given to the active validators (per round) after giving the minimum reward. For the Lisk mainchain, as calculated in the [`table`](#reward-computation), the rewards are `90.9*10^8` beddows or `90.9 LSK`. This way, the module achieves fairness (higher weight implies higher rewards) while ensuring that rewards of low-weight validators do not go below the threshold of profitability.
-* In principle, if percentage of rewards shared equally among active validators is set to `0%` i.e.`FACTOR_MINIMUM_REWARD_ACTIVE_VALIDATORS == 0` (as described in [table](#specification)), then it will be a proof-of-stake system where rewards are proportional to the validator weight. On the other hand, if the percentage of rewards shared equally among active validators is set to `100%` i.e. `FACTOR_MINIMUM_REWARD_ACTIVE_VALIDATORS == 10000`, it will be the previous proof-of-stake (PoS) system where each validator received the same reward for forging a block irrespective of their weights. Therefore, this reward system can be seen as a generalization of the previous one and allows flexibility to developers to use the variant of Proof-of-Stake they find more appropriate for their use case.
+* This module distributes the remaining stake rewards i.e. `getStakeRewardActiveValidators(blockHeader.height)` proportional to the validators weight. It is given to the active validators that generate a block, on top of the minimum reward. For the Lisk mainchain, the rewards distributed according to stake are 90% of total rewards, therefore `90.9*10^8` beddows or `90.9 LSK` per round (assuming `101` active validators). This way, the module achieves fairness (higher weight implies higher rewards) while ensuring that rewards of low-weight validators do not go below the threshold of profitability.
+* In principle, if percentage of rewards shared equally among active validators is set to `0%` i.e.`FACTOR_MINIMUM_REWARD_ACTIVE_VALIDATORS == 0`, then it will be a proof-of-stake system where rewards are proportional to the validator weight. On the other hand, if the percentage of rewards shared equally among active validators is set to `100%` i.e. `FACTOR_MINIMUM_REWARD_ACTIVE_VALIDATORS == 10000`, it will be the previous proof-of-stake (PoS) system where each validator received the same reward for forging a block irrespective of their weights. Therefore, this reward system can be seen as a generalization of the previous one and allows flexibility to developers to use the variant of Proof-of-Stake they find more appropriate for their use case.
```

Comment:
The comma is not fixed. In general, it would make the review process easier if you accept suggestions on GitHub (can also be done in one batch) instead of doing the same change again yourself.

Reply:
Fixed.
Maybe I should try this gh tool, I just thought that gathering changes from the web UI and locally would be more complicated than doing all in a single commit.